src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -1824,6 +1824,12 @@ public void SaveImage(object parameters)
                 "NodeCount", CurrentSpace.Nodes.Count());
         }
 
+        public void Save3DImage(object parameters)
```

Comment:
if we're adding this to the public API should document it especially what goes in this parameter